src/app/pages/AddPage/AddPage.tsx

```diff
@@ -26,50 +31,51 @@ export default function AddPage(): JSX.Element {
     };
   });
 
-  const allCategorys = skills?.map((skills) => {
-    return {
-      title: skills.title,
-      imageSrc: skills.imageSrc,
-      description: skills.description,
-      category: skills.category,
-    };
+  const allCategorys = skills?.map((skill) => {
+    const { title, imageSrc, description, category, id, progress, isDone } =
+      skill;
+    return { title, imageSrc, description, category, id, progress, isDone };
   });
   const filteredSkills = allCategorys?.filter(
     (skills) => skills.category === selectedCategory
   );
 
+  function handleAddOnClick(skills: Skill) {
+    addOnClick(skills);
+  }
+
   return (
-    <>
-      <div className={styles.container}>
-        <header className={styles.header}>
-          <Header
-            className={styles.head}
-            type="backwithimage"
-            imageSrc="https://images.unsplash.com/photo-1522075469751-3a6694fb2f61?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=100&q=80"
-            onClick={() => {
-              history.push('/');
-            }}
-          />
-          <TagGroup tagList={tagList} className={styles.tagGroup} />
-        </header>
-        <main className={styles.cardWrapper}>
-          {filteredSkills &&
-            filteredSkills.map((skills) => {
-              return (
-                <Card
-                  type="add"
-                  {...skills}
-                  imageSrc={skills.imageSrc}
-                  title={skills.title}
-                  description={skills.description}
-                />
-              );
-            })}
-        </main>
-        <div>
-          <Navigation activeLink={'add'} />
-        </div>
+    <div className={styles.container}>
+      <div className={styles.header}>
+        <Header
+          className={styles.head}
+          type="backwithimage"
+          imageSrc="https://images.unsplash.com/photo-1522075469751-3a6694fb2f61?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=100&q=80"
+          onClick={() => {
+            history.push('/');
+          }}
+        />
+        <TagGroup tagList={tagList} className={styles.tagGroup} />
       </div>
-    </>
+      <main className={styles.cardWrapper}>
+        {filteredSkills &&
+          filteredSkills.map((skills) => {
+            return (
+              <Card
+                {...skills}
+                id={skills.id}
+                progress={skills.progress}
+                type="add"
+                imageSrc={skills.imageSrc}
+                title={skills.title}
+                description={skills.description}
+                onClick={() => handleAddOnClick(skills)}
```

Comment:
Just some light name changes could make this read like: `onAddClick={() => addSkill(skill)}`